Mathlib/LinearAlgebra/PiTensorProduct.lean

```diff
@@ -678,6 +680,230 @@ this is false in the case where `ι` is empty. -/
 instance : AddCommGroup (⨂[R] i, s i) :=
   Module.addCommMonoidToAddCommGroup R
 
+/-!
+#### Tensor product of `R`-algebras
```

Comment:
I think this should go in `RingTheory.PiTensorProduct` to match `RingTheory.TensorProduct`.